config/main.py

```diff
@@ -876,14 +1052,42 @@ def ip(ctx):
 
 @ip.command()
 @click.argument("ip_addr", metavar="<ip_addr>", required=True)
+@click.argument('gw1', metavar='<default gateway IP address>', required=False)
 @click.pass_context
-def add(ctx, ip_addr):
+def add(ctx, ip_addr, gw1):
     """Add an IP address towards the interface"""
     config_db = ctx.obj["config_db"]
     interface_name = ctx.obj["interface_name"]
 
     if interface_name.startswith("Ethernet"):
         config_db.set_entry("INTERFACE", (interface_name, ip_addr), {"NULL": "NULL"})
+    elif interface_name.startswith("eth0"):
```

Comment:
@pavel-shirshov 
pavel-shirshov 11 days ago Contributor 
It could match interface 'eth01'. I think it's better to compare to 'eth' here

Will address this.

Reply:
@pavel-shirshov : Addressed your comment. Changed "startswith eth0", to "exact match for eth0"